src/main/java/racingcar/view/constants/ConstantMessage.java

```diff
@@ -0,0 +1,21 @@
+package racingcar.view.constants;
+
+public enum ConstantMessage {
+    // Request
+    ASK_MULTIPLE_CAR_NAMES("경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)"),
```

Comment:
만약, 경주할 자동차의 이름을 쉼표가 아닌 공백으로 바뀐다면 InputValidator, Parser, ConstantMessage... 등등을 함께 바꾸어 주어야 하나요?  그렇다면 따로 관리해보는 건 어떨까요!?

Reply:
@le2sky 
저도 이 부분을 생각했었습니다! 
코드에 미처 적용하지 못했지만 `"쉼표"`와 `","`도 하나의 필드로 선언해서
`ASK_MULTIPLE_CAR_NAMES`의 문자열에 넣어줄 것 같아요😉

```suggestion
    SEPERATOR("쉼표"),
    ICON(","),
    ASK_MULTIPLE_CAR_NAMES("경주할 자동차 이름을 입력하세요.(이름은 " + SEPERATOR + "(" + ICON + ") 기준으로 구분)"),
```

Reply:
이전 과제에서 정확히 그렇게 설계한 경험이 있습니다.
![스크린샷 2023-11-02 오전 3 17 13](https://github.com/woowacourse-precourse/java-racingcar-6/assets/112257466/b549be51-eb7e-4634-a54b-fb4514e056e2)

숫자야구 게임에서 숫자의 길이와, 제한을 자유롭게 조절하고, 이에 따라 결과도 자연스럽게 나타날 수 있도록 설계했어요.

> 🌱 0x05 Open-Closed Principle vs YAGNI
> 실제로 필요할 때 무조건 구현하되, 그저 필요할 것이라고 예상할 때에는 절대 구현하지 말라.

출처 : [YAGNI](https://ko.wikipedia.org/wiki/YAGNI) - Wikipedia

기존 1주차에 OCP를 준수하기 위해 작성했던 코드에서, 과제 요구사항과는 거리가 있다는 리뷰를 받았습니다.
반면 긍정적인 리뷰도 있었습니다. 확장성 있는 설계로 좋은 리뷰를 남겨주신 분도 계셨습니다.
이번 미션은 자동차 경주 게임의 본질을 꾸준히 되새기며 YAGNI 설계로 구성했습니다.

> 실제로 필요할 때 무조건 구현하되, 그저 필요할 것이라고 예상할 때에는 절대 구현하지 말라.

Reply:
아하.. 그런 맥락이 있었군요! 자세한 맥락 설명 감사드립니다! 저도 평소에 YAGNI를 생각하면서, 프로젝트 개발에 임하는데요. 갑자기 재미난 생각이 들어서, 해빈님 의견을 듣고 싶습니다!

위키에서는 `익스트림 프로그래밍은 비즈니스 상의 요구가 시시각각 변동이 심한 경우에 적합한 개발 방법이다.` 라고 적혀 있더군요! 그리고 YAGNI가 익스트림 프로그래밍의 원칙 중 하나라면, 자동차 경주 게임과 같은 코드 카타에서도 YAGNI를 신경 쓰는게 맞을까요?

Reply:
@le2sky

어떤 설계방법이 우아한테크코스에 적합하다는 질문에는 확실하게 답변을 드리기 어려울 것 같아요!
다만, 주차별 과제를 진행함에 있어, (사실상 수정 하지 않을 부분들까지) 확장성을 너무 염두해 둔 탓에
1주차는 다른 부분에 신경쓸 시간이 줄어들어서 시간을 오래 투자하지 못했던 부분이 아쉬웠습니다!
개발자에게 가장 중요한 부분은 데드라인 준수니까요!

어쩌면 7일이라는 짧은 기간동안 그 모든 것을 준수하고 작성하는 OCP 설계가 가장 이상적이겠지만,
모든 부분을 확장으로 염두해두고 설계하는 것 자체는 본 과제에서 요구하고, 함양시키고자 하는
프로그래밍 기본 능력과는 조금 거리가 있다고 판단했습니다!
그래서 이번 과제에서는 XP의 YAGNI 원칙에 입각해, 설계해 보았습니다 :)